client/webserver/site/src/html/forms.tmpl

```diff
@@ -15,29 +15,31 @@
     <input type="text" class="form-control select" id="acctName">
   </div>
   <div>
-    <label for="newWalletPass" class="pt-3 pl-1 mb-1">Wallet Password</label>
-    <input type="password" autocomplete="off" class="form-control select" id="newWalletPass">
+    <label for="newWalletPass" class="pt-3 pl-1 mb-0">Wallet Password</label>
+    <div class="fs14 px-1 mb-1">This is the password you have configured with your wallet backend.</div>
+    <input type="password" class="form-control select" id="newWalletPass" autocomplete="off">
   </div>
   <div>
     <label for="iniPath" class="pt-3 pl-1 mb-1">Configuration Filepath</label>
     <input type="text" class="form-control select" placeholder="leave empty to use default path" id="iniPath">
   </div>
   <div>
-    <label for="wClientPass" class="pt-3 pl-1 mb-0">Client Password</label>
-    <input type="password" class="form-control select" id="wClientPass" autocomplete="off">
+    <label for="nwClientPass" class="pt-3 pl-1 mb-0">Client Password</label>
+    <div class="fs14 px-1 mb-1">Your client password is always required when performing sensitive wallet operations.</div>
```

Comment:
This message in particular is intended for brand new users going through the registration page. I don't think it's great to display this on this form every time. That's the reason I wasn't using the `newWalletForm` template on the registration page. That page is intended to guide users through some basic DEX concepts.

Reply:
I see. To keep this code DRY as much as possible, I want to add a template property that controls whether or not to display the help texts.